addons/common/functions/fnc_showHud.sqf

```diff
@@ -39,7 +39,7 @@ if (isArray (missionConfigFile >> "showHUD")) then {
 };
 
 if (_reason != "") then {
-    _reason = toLower _reason;
+    _reason = toLowerANSI _reason;
```

Comment:
I don't think it's a good idea here either.

Reply:
I *think* hashmap keys don't support non-ANSI characters either

Reply:
yeah they do, gonna revert this